<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>-1</string>
    </property>
    <property name="text">
     <string>LEFT</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">leftrightbuttons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>400</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>1</string>
    </property>
    <property name="text">
     <string>UP</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">updownbuttons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>450</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>1</string>
    </property>
    <property name="text">
     <string>RIGHT</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">leftrightbuttons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>450</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>-1</string>
    </property>
    <property name="text">
     <string>DOWN</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">updownbuttons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>450</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>0.5</string>
    </property>
    <property name="text">
     <string>PgDown</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">pgbuttons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>400</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>2</string>
    </property>
    <property name="text">
     <string>PgUp</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">pgbuttons</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>map</string>
    </property>
    <property name="text">
     <string>Cхема</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">switchertypemap</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>70</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>sat</string>
    </property>
    <property name="text">
     <string>Спутник</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">switchertypemap</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>90</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>sat,skl</string>
    </property>
    <property name="text">
     <string>Гибрид</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">switchertypemap</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Тип карты:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="gps_lbl">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>150</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="searchbutton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>180</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Искать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>120</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Искать объект по названию:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>180</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Сбросить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>400</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Адрес запрошеного объекта:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>250</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>1</string>
    </property>
    <property name="text">
     <string>Включено</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">switcherpostcode</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_5">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>270</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>0</string>
    </property>
    <property name="text">
     <string>Выключено</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">switcherpostcode</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>220</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Отображение почтового индекса:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="adress_lbl">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>430</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="postcodelbl">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>480</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>813</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="switcherpostcode"/>
  <buttongroup name="switchertypemap"/>
  <buttongroup name="pgbuttons"/>
  <buttongroup name="leftrightbuttons"/>
  <buttongroup name="updownbuttons"/>
 </buttongroups>
</ui>
